app/src/main/res/layout/activity_article_create.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:id="@+id/content">
+
+    <android.support.design.widget.TextInputLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <android.support.design.widget.TextInputEditText
+            android:id="@+id/article_name_field"
+            android:layout_width="match_parent"
+            android:layout_height="50dp"
+            android:hint="@string/article_title"
+            android:inputType="text"
+            android:lines="1"
+            android:textSize="@dimen/text18" />
+    </android.support.design.widget.TextInputLayout>
+
+    <android.support.design.widget.TextInputLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <android.support.design.widget.TextInputEditText
+            android:id="@+id/article_content_field"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/acticle_content"
+            android:inputType="text"
+            android:lines="1"
+            android:textSize="@dimen/text18" />
+    </android.support.design.widget.TextInputLayout>
+
+    <Button
```

Comment:
Let's be consistent. Add save icon in menu such as in https://github.com/vycius/NMAkademija/blob/master/app/src/main/res/menu/edit_profile_menu.xml#L4